connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -25,13 +36,24 @@ class EmbeddedComponentManager(
     private val _appearanceFlow = MutableStateFlow(appearance)
     internal val appearanceFlow: StateFlow<Appearance> get() = _appearanceFlow.asStateFlow()
 
+    internal suspend fun requestCameraPermission(context: Context): Boolean? {
+        val activity = context.findActivity() ?: error("You must create an AccountOnboardingView from an Activity")
+        launcherMap[activity]?.launch(android.Manifest.permission.CAMERA) ?: return null
```

Comment:
I agree it's a worse bug to crash than it is to let the user proceed but with broken permissions. What do you think about crashing if in debug (since this really is a programmer error), but then following the behavior you outlined otherwise?